src/components/atoms/Text.tsx

```diff
@@ -19,6 +21,12 @@ export const Text: FunctionComponent<IProps> = ({type, children}) => {
       return (<h2>{children}</h2>);
     
     case TextType.CONTENT:
-      return (<p>{children}</p>)
+      return (<p>{children}</p>);
```

Comment:
(minor)
If `<p>{children}</p>` is a default or a placeholder, then consider doing one of these instead:
- Move `<p>{children}</p>` to a constant that explains that this is a `DEFAULT_LAYOUT` or a `PLACEHOLDER_LAYOUT`.
- Use a list of 'if' instead of a 'switch' and use a single `if` for all of these pages.
- Use a `default` statement.